yarn.lock

```diff
@@ -158,7 +158,7 @@ __metadata:
     fs-extra: ^8.1.0
     ganache-core: ^2.10.1
     http-proxy: ^1.18.0
-    ipfs-http-client: ^42.0.0
+    ipfs-http-client: 43.0.0
```

Comment:
Please add a brief explanation regarding the strict `ipfs-http-client` version pinning :pray:

Reply:
Added https://github.com/lidofinance/lido-dao/pull/436/commits/0f64bf1b0df6a2bb5ba2335718453baad9feab71